runtime/runtime/rtc.go

```diff
@@ -1,3 +1,5 @@
+//go:build encore_internal
```

Comment:
I'd say we should revert these tags and add them in later when we figure out how to merge the `encore.dev` module into the runtime